go/vt/vtctl/workflow/traffic_switcher.go

```diff
@@ -1490,9 +1493,76 @@ func (ts *trafficSwitcher) getTargetSequenceMetadata(ctx context.Context) (map[s
 		return nil, err
 	}
 
-	if tablesFound != tableCount {
-		return nil, vterrors.Errorf(vtrpcpb.Code_INTERNAL, "failed to locate all of the backing sequence tables being used: %s",
-			strings.Join(maps.Keys(sequencesByBackingTable), ","))
+	if len(tablesFound) != tableCount {
```

Comment:
This can be extracted into a separate function.

Reply:
I moved it out here: https://github.com/vitessio/vitess/pull/16860/commits/eaed239217eae04f60f3734b0cf35ced179e7cd9